_posts/2017-10-03-continuous-integration-in-github.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: post
+title: Continuous Integration in GitHub
+date: 2017-10-03
+---
+
+I have been working on a project for the last couple of days and I decided it was about time I set up some kind of Continuous Integration tool to my repo in GitHub. I will describe here how you can achieve that in a few minutes with Travis CI.
+
+First of all, what is Continuous Integration?
+
+> Continuous integration is a DevOps software development practice where developers regularly merge their code changes into a central repository, after which automated builds and tests are run. Continuous integration most often refers to the build or integration stage of the software release process and entails both an automation component (e.g. a CI or build service) and a cultural component (e.g. learning to integrate frequently). The key goals of continuous integration are to find and address bugs quicker, improve software quality, and reduce the time it takes to validate and release new software updates.
+<br />
+> [AWS](https://aws.amazon.com/devops/continuous-integration/)
```

Comment:
Is this right? The '>' means the link/text doesn't render properly

Reply:
weird, works when I run it locally. Hmm... Ignore it for now pls.